bucket/wasmer.json

```
@@ -8,24 +8,23 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://github.com/wasmerio/wasmer/releases/download/v4.3.3/wasmer-windows.exe",
-            "hash": "8f631b6ae787bd7e2a7fa7feeea3ad662636482fa63f6eeac89b9ea353184708"
+            "url": "https://github.com/wasmerio/wasmer/releases/download/v4.3.3/wasmer-windows-amd64.tar.gz",
+            "hash": "63239253e7dc5373a421941aaa9ac8ee2ddb2f34e3d94b12ea739319d7929bd4"
         }
     },
     "env_set": {
         "WASMER_DIR": "$dir",
         "WASMER_CACHE_DIR": "$dir\\cache"
     },
     "bin": "bin\\wasmer.exe",
-    "innosetup": true,
     "checkver": {
         "github": "https://github.com/wasmerio/wasmer",
         "regex": "tag/([\\w.-]+)"
```

```suggestion
        "github": "https://github.com/wasmerio/wasmer"
```
It would be better to remove `regex` section. That's because currently there is unnecessary `v` in wasmer's version information. In default, Scoop recognizes the new version from releases automatically without `v`.

---

Removed, updated to the latest version, and rebased